### ISSUE TITLE
Linux client does not honor selection buffer

### ISSUE THREAD

**Issue by phaoost** [telegramdesktop/tdesktop#2072]
### Steps to reproduce
1. Open any chat in telegram.
2. Select any text with pointing device left button
3. Open any other destination window and click middle button on text input area.

### Expected behaviour
The marked text should appear in destination window

### Actual behaviour
Nothing happens. We have to use other sequence: mark, right-click, choose copy selected text. On destination window we need to press ctrl-v or right-click and choose paste.

### Configuration
Linux with KDE

**Version of Telegram Desktop:**
0.9.49

Please fix normal linux behavior.

**Comment by loentar** [comment]
Select and middle-click-paste is one of most used features in Linux every day. I very rare use Ctrl+Ins (Ctrl+C) and Shift+Ins (Ctrl+V) to copy-paste text because it's much slower than just select a text by mouse and paste it with middle click into destination window. It's just 2 buttons to be pressed (left click to select, middle to paste) instead of 6 (left click select, Ctrl, C, left click to destination edit box, Ctrl, V)!

It's very inconvenient that selection from chat area cannot be pasted into chat line and into any other window as well (but select and paste currently typing message works).

To fix it you just need to add this line after selection finished:

```C++
    QApplication::clipboard()->setText(selectedText, QClipboard::Selection);
```

For additional info please see:
http://doc.qt.io/qt-5/qclipboard.html#notes-for-x11-users
https://burrows.svbtle.com/universal-copy-paste-in-linux

**Comment by john-preston** [comment]
@loentar Thanks! I'll try add it in one of the future versions.

**Comment by loentar** [comment]
This seems to be working now in `0.9.50`.
And it was resolved in 38c291553366b9ed1bf5cec52a2cc1bdd4b2608b.

Thank you very much!
